config/initializers/bower_rails.rb

```diff
@@ -0,0 +1,16 @@
+BowerRails.configure do |bower_rails|
```

Comment:
Unused block argument - `bower_rails`. You can omit the argument if you don't care about it.